definitions/npm/metalsmith-layouts_v2.x.x/flow_v0.75.x-/metalsmith-layouts_v2.x.x.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-unused-vars, no-redeclare */
+
+declare module 'metalsmith-layouts' {
+
+  import type { Plugin } from 'metalsmith';
```

Comment:
This won't work, discussed in contributing doc.

Reply:
I see now. flow-typed still does not support importing type definitions from other libdefs. That's really silly, I'd need to copy&paste basically complete type definitions file from metalsmith to write typedefs for metalsmith's plugins. I don't wanna do that.

Reply:
Yeah, we're all pretty bummed about it. #flowStruggles.